.travis.yml

```diff
@@ -23,17 +22,12 @@ notifications:
     on_success: change
     on_failure: change
 before_script:
-  - mysql -u root -e 'create database osem_test;'
   - RAILS_ENV=test bundle exec rake db:bootstrap --trace
 script:
   - "./travis_script.sh $TEST_SUITE"
 env:
   global:
-    - OSEM_DB_ADAPTER=mysql2
-    - OSEM_DB_USER=root
-    - OSEM_DB_PASSWORD=''
-    - OSEM_DB_HOST=''
-    - OSEM_DB_PORT=''
+    - OSEM_DB_ADAPTER=sqlite3
```

Comment:
Phew this is very far away from any production instance I know. Might be a temporary fix, but not a solution.

Reply:
Well, that's supposed to be the point of ActiveRecord, that the database behind the ORM doesn't matter. If we're worried about full-stack production testing, we'd need to run parallel on MySQL & Postgres. IMO the only thing interim about this as a test db, is that we're not running in-memory. Yet.

Reply:
Not to be argumentative, but just to be clear: the `osem` code didn't change, but now the tests pass. This is a pretty clear indicator that the interaction of capybara+database_cleaner+mysql is the culprit for our flaky tests, not the test suite itself.

Reply:
> capybara+database_cleaner+mysql is the culprit for our flaky tests,

Nobody debates this.

Reply:
Define `good` :wink: I fear we are going to stay with this a long time. Because it's so convenient...